### PR TITLE
metrics: register skipper_filter_create_duration_seconds

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Follow up on #3115

### CODE REVIEW

File: metrics/prometheus_test.go [hunk 1]
```diff
@@ -73,6 +73,29 @@ func TestPrometheusMetrics(t *testing.T) {
 			},
 			expCode: http.StatusOK,
 		},
+		{
+			name: "Filter creation latency",
+			addMetrics: func(pm *metrics.Prometheus) {
+				pm.MeasureFilterCreate("filter1", time.Now().Add(-15*time.Millisecond))
```
Comment (RomanZavodskikh) [inline metrics/prometheus_test.go:79]: Maybe add some second value, -time.Second, for example?

Comment (AlexanderYastrebov) [reply metrics/prometheus_test.go:79]: It does not matter, we don't test that prometheus works correctly but rather that metric is registered.

**Comment by MustafaSaber** [comment]
👍

**Comment by RomanZavodskikh** [comment]
:+1:

**Comment by szuecs** [comment]
👍

**Pull request merged**